### DISCUSSION TITLE
VS code's inbuilt simple browser fails to load?

### DISCUSSION THREAD

**Discussion by Blakeinstein** [coder/code-server#3844] [Q&A]
I tried initially on a docker container. But can reproduce this on the VM version aswell.
On the vm version some files fail due to CORS? (No cors error seen on the docker version)

### Steps to reproduce
1. Open Command Palette (Ctrl/Cmd + shift + p / F1)
2. Type Simple Browser
3. Select Simple Browser(show)
4. Type any url
5. (optional) check browser console for errors

**Comment by jsjoeio** [comment]
Yeah, I can't get this to work running code-server locally on macOS + Brave. This must be a bug. I'll open a report.

![image](https://user-images.githubusercontent.com/3806031/126841461-ea55229a-40c6-43dc-a38a-a7c51050c28d.png)

**Reply by jsjoeio** [reply]
Thank you for flagging!

**Reply by jsjoeio** [reply]
https://github.com/cdr/code-server/issues/3845
